### PR TITLE
Admin importer fixes for #381

### PR DESCRIPTION
**Pull request by rstorey**

### CODE REVIEW

File: concordia/admin.py [hunk 1]
```diff
@@ -103,11 +103,12 @@ def admin_bulk_import_view(request):
                 import_url_blob = row["Import URLs"]
 
                 if not all((campaign_title, project_title, import_url_blob)):
-                    messages.add_message(
-                        request,
-                        messages.WARNING,
-                        f"Skipping row {idx}: at least one required field (Campaign, Project, Import URLs) is empty",
+                    warning_message = (
+                        f"Skipping row {idx}: at least one required field \
```
Comment (acdha) [inline concordia/admin.py]: Putting the line continuation in there puts all of the subsequent indentation into the string. I'd just split this into two implicitly-concatenated strings:

```suggestion
                        f"Skipping row {idx}: at least one required field" \
```

File: concordia/admin.py [hunk 2]
```diff
@@ -103,11 +103,12 @@ def admin_bulk_import_view(request):
...
+                        (Campaign, Project, Import URLs) is empty",
```
Comment (acdha) [inline concordia/admin.py]:
```suggestion
                        " (Campaign, Project, Import URLs) is empty",
```

File: concordia/admin.py (hunk 1 above)
Comment (rstorey) [reply concordia/admin.py]: Done

**Comment by rstorey** [comment]
See #381

File: concordia/admin.py [hunk 3]
```diff
@@ -30,6 +30,46 @@
...
 
+def publish_item_action(modeladmin, request, queryset):
+    """
+    Mark all of the selected items and their related assets as published
+    """
+
+    count = queryset.filter(published=False).update(published=True)
+    asset_count = 0
+    for item in queryset:
+        asset_count += Asset.objects.filter(item=item, published=False).update(
```
Comment (acdha) [inline concordia/admin.py]: I think we could avoid the loop by making this `asset_count += Asset.objects.filter(item__in=queryset, …)`

Comment (rstorey) [reply concordia/admin.py]: I was going to ask if there was a way to avoid the loop. Thanks

**Pull request merged**